### PR TITLE
Add mentor details columns to the mentor table

### LINKED ISSUE #218: Add mentor details columns to the mentor table
**Is your feature request related to a problem? Please describe.**
For the February edition of scholarx, we are adding a few more questions to the mentor application. And we need to store the responses in the mentor table.

**Describe the solution you'd like**
Add the following fields to the Mentor Model

1. Category (category)
2. Area(s) of Expertise (expertise)
3. Institution (institution)
4. Position (position)
5. Bio (bio)
6. No. of Mentee Slots (slots)

https://sef.discourse.group/t/scholarx-february-edition/359/11?u=piumal1999

**Describe alternatives you've considered**
N/A

**Additional context**
We can keep categories as enums too. But for that, we'll need to define the categories we have. (Engineering, Life Sciences, Computer Science, Data Science & AI, Physical Science, Other)

### PR DESCRIPTION
**Pull request by Gimhan-minion**
## Purpose
The purpose of this PR is to fix #218 

## Goals
To add new mentor details to the model class

## Approach
Add the following fields to the Mentor Model

- Category (category)
- Area(s) of Expertise (expertise)
- Institution (institution)
- Position (position)
- Bio (bio)
- No. of Mentee Slots (slots)

### Screenshots
image
(Screenshot of generated mentor table)

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/main/java/org/sefglobal/scholarx/model/Mentor.java [hunk 1]
```diff
@@ -16,9 +18,75 @@
     public Mentor() {
     }
 
+    @Column
```
Comment (Piumal1999) [inline src/main/java/org/sefglobal/scholarx/model/Mentor.java:21]:
```suggestion
    @Enumerated(EnumType.STRING)
    @Column(nullable = false)
```

**Review by Piumal1999 — CHANGES_REQUESTED** [review]
Looks good. Could you please run the backend and get a screenshot of generated mentor table.

File: src/main/java/org/sefglobal/scholarx/model/Mentor.java (hunk 1 above)
Comment (Gimhan-minion) [reply src/main/java/org/sefglobal/scholarx/model/Mentor.java]: @Piumal1999 Made the corresponding changes in the code. Also updated a screenshot in the PR iteself.

Comment (Gimhan-minion) [reply src/main/java/org/sefglobal/scholarx/model/Mentor.java]: Also are there any fields that should be not null? Or is this fine?

Comment (Piumal1999) [reply src/main/java/org/sefglobal/scholarx/model/Mentor.java]: I think it's fine

**Review by Piumal1999 — APPROVED** [review]
Good Job! @Gimhan-minion

**Pull request merged**